### DISCUSSION TITLE
Faster Optimization

### DISCUSSION THREAD

**Discussion by efeint01** [kernc/backtesting.py#1016] [Q&A]
Hi.

I work on BTCUSDT 1 minute interval 3 years data. And using **"skopt"** optimization **max_tries=200**. I need optimize **16** parameters.

It takes a lot of time. Around -45Minute for 1 backtesting with this huge data. And gave me really bad results is not worth the time.

Is there anything can we speed up things like using parallel for skopt optimization?
Im using Windows 11. I read the this article: https://kernc.github.io/backtesting.py/doc/examples/Parameter%20Heatmap%20&%20Optimization.html#Model-based-optimization

**Comment by PilotGFX** [comment]
I suggest installing ubuntu, either as a typical install if you like, or with virtual box and run it through there. it seems that the lib is made to work using fork and spawn is not working properly.  and also i think you will be better off only optimizing on 3-6 paramters.
